src/eom-properties-dialog.c

```diff
@@ -591,13 +568,121 @@ eom_properties_dialog_class_init (EomPropertiesDialogClass *class)
 							      FALSE,
 							      G_PARAM_READWRITE |
 							      G_PARAM_STATIC_STRINGS));
+
+	gtk_widget_class_set_template_from_resource ((GtkWidgetClass *) klass, "/org/mate/eom/ui/eom-image-properties-dialog.ui");
```

Comment:
You could put this line below the next, that way you don't need to cast `klass` twice and just reuse `wklass`

Reply:
Thanks and done.